test/unittest/dds/participant/CMakeLists.txt

```diff
@@ -17,9 +17,8 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/../profiles/test_xml_profiles.xml
     COPYONLY)
 
 set(PARTICIPANTTESTS_SOURCE
-    ParticipantTests.cpp
-    ${PROJECT_SOURCE_DIR}/src/cpp/rtps/builtin/discovery/participant/ServerAttributes.cpp
```

Comment:
This makes the windows compilation fail as 
`template<> const char* server_ostream_separators<char>::locator_separator = "|";` 
is not exported.

Reply:
Also, linters are failing (uncrustify).

Kudos for resolving the fastbins memory corruption :partying_face: and I am still trying to stop laughing about the :crying_cat_face: poor kitten crying :laughing: